inc/classes/class_translation.php

```diff
@@ -393,7 +393,7 @@ public function xml_read_to_array($module) {
      * @param string    $module     Module name (System, DB, Module ...)
      * @return string
      */
-    public function get_trans_filename($module)
+    private function get_trans_filename($module)
```

Comment:
Referened in TranslationTest.php L160

Reply:
Deleted in this PR also: https://github.com/lansuite/lansuite/pull/190/files#diff-ce67f17604206546cec768ee5f8b7802L158